test/api/assignments_test.rb

```diff
@@ -75,8 +76,9 @@ def test_api_accepts_submission_on_completed_exercise
     Notify.stub(:everyone, nil) do
       Xapi.stub(:exists?, true) do
         solution = { "THE CODE" => "go/one/code.go" }
-        post '/user/assignments', { key: alice.key, code: 'THE CODE', path: 'go/one/code.go', 
-          'language' => 'go', 'problem' => 'one', solution: solution }.to_json
+        post '/user/assignments', { key: alice.key, code: "THE CODE", path: "go/one/code.go",
+                                  "language" => "go", "problem" => "one", 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.